lib/api/multipartDelete.js

```diff
@@ -148,6 +150,18 @@ function multipartDelete(authInfo, request, log, callback) {
                         keys: [objectKey],
                         byteLength: partSizeSum,
                     });
+                    if (location) {
+                        // partSizeSum passed as negative to decrement metric
+                        return pushLocationMetric(location, -partSizeSum, log,
```

Comment:
Not sure, but isn't this already done at line 118 (`data.delete()` which in turns calls the backend `_retryDelete` which decrements the metric as well)?

Reply:
Yes! Good catch 😅